core/engine/src/module/mod.rs

```diff
@@ -223,6 +223,19 @@ impl Module {
         )
     }
 
+    /// Create a module that exports a single JSON value as the default export, from its
+    /// JSON string. This required the `json` feature to be enabled.
+    ///
+    /// # Errors
+    /// This will return an error if the JSON string is invalid or cannot be converted.
```

Comment:
Note: discuss whether this should be gated behind the experimental feature or not. I would say no because it's only accessible by embedders, and import attributes should be the ones gated.

Reply:
Yeah, this is just a note for future reference.